README.md

```diff
@@ -29,9 +29,15 @@ Version 2 of the Envoy API will evolve existing APIs and introduce new APIs to:
 
 ## Status
 
-Draft work-in-progress. Input is welcome via issue filing. Small, localized PRs
-are also welcome, but any major changes or suggestions should be coordinated in
-a tracking issue with the authors.
+The LDS/CDS/EDS/RDS APIs are now frozen and will maintain backwards
```

Comment:
Yes. This isn't super urgent for us (Lyft) since we really only need RDS frozen. If you need to make structural changes to support SNI (which I think you do), can you potentially front load that?